docs/development/setup-vagrant.md

```diff
@@ -566,6 +566,21 @@ When reporting your issue, please include the following information:
 The output of `tools/diagnose` run inside the Vagrant guest is also
 usually helpful.
 
+#### Vagrant Gem Not found(for Ubuntu/Debian based System)
+If you get an error like
+```
+Traceback (most recent call last):
+2: from /usr/bin/vagrant:23:in <main>'
+    1: from /usr/lib/ruby/2.7.0/rubygems.rb:294:in activate_bin_path'
+/usr/lib/ruby/2.7.0/rubygems.rb:275:in `find_spec_for_exe': can't find gem vagrant (>= 0.a) with executable vagrant (Gem::GemNotFoundException)
+```
+Then vagrant wasn't installed properly on your system.This happens if you have
```

Comment:
Then vagrant wasn't installed properly on your system-> It means that vagrant is not installed properly.